scripts/subctl/getsubctl.sh

```diff
@@ -0,0 +1,111 @@
+#!/bin/bash
+set -e
+get_operating_system() {
+  case $(uname -s) in
+
+    Darwin*) echo "darwin" ;;
+    Linux*) echo "linux" ;;
+    CYGWIN*) echo "windows" ;;
+    MINGW*) echo "windows" ;;
+    *)
+    error "This installer is only supported on Linux and macOS and Windows. Found $(uname -s)"
+    return 1;;
+  esac
+}
+
+get_architecture() {
+  case $(uname -m) in
+
+    x86_64) echo "amd64" ;;
+    amd64) echo "amd64" ;;
+    i386) echo "386" ;;
+    i686) echo "386" ;;
+    *)
+    error "This installer is only for x86_64 and i386 architectures. Found $(uname -m)"
+    return 1;;
+  esac
+}
+
+command_exists() {
+  command -v "$@" > /dev/null 2>&1
+}
+
+download_command() {
+  if command_exists curl; then
+    echo "curl -fsSL"
+  elif command_exists wget; then
+    echo "wget -qO-"
+  else
+    error "curl and wget are missing"
+    return 1
+  fi
+}
+
+GET=$(download_command)
+
+get_subctl_release_url() {
+  version=$1
+  os=$2
+  architecture=$3
+
+  case ${version} in
+    latest) url=https://api.github.com/repos/submariner-io/submariner-operator/releases/latest ;;
+    master) url=https://api.github.com/repos/submariner-io/submariner-operator/releases/tags/latest ;;
+    *) url=https://api.github.com/repos/submariner-io/submariner-operator/releases/tags/${version} ;;
+  esac
+
+  ${GET} "${url}" | grep "browser_download_url.*-${os}-${architecture}" | cut -d\" -f 4
+
+}
+
+get_subctl() {
+  version=$1
+  architecture=$2
+  os=$3
+  url=$4
+
+  binary_name="subctl-${version}-${os}-${architecture}"
+
+  tmpdir=$(mktemp -d)
+  cd "${tmpdir}"
+
+  case ${url} in
+    *tar.xz)
+        ${GET} "${url}" > "${binary_name}.tar.xz"
+        tar xvf "${binary_name}.tar.xz"
+        ;;
+    *) # non tar.xz releases (older)
+      mkdir -p "${binary_name}"
+      ${GET} "${url}" > "${binary_name}/${binary_name}"
+      ;;
+  esac
+
+  install_subctl "${binary_name}/${binary_name}"
+}
+
+install_subctl() {
+  bin=$1
+  destdir=~/.local/bin
+  dest=${destdir}/subctl
+
+  mkdir -p "${destdir}"
+  cp -f "${bin}" "${dest}"
```

Comment:
```suggestion
  install "${bin}" "${dest}"
```

to ensure the command ends up executable.